docs/pages/mermaid.md

```diff
@@ -24,8 +24,6 @@ diagrams dynamically*. The syntax is easy and clear, and the output usually beau
 
 See [mermaid-js#integrations](https://mermaid-js.github.io/mermaid/#/integrations?id=integrations) for a full list.
 
-**GitHub** does not support it for now, but there are many browser extensions available
-that will automatically detect and render the diagrams in md files.
-I personally use [mermaid-extension](https://github.com/Stefan-S/mermaid-extension).
+**GitHub** does support it, too
 
-[Enable Mermaid rendering in GitHub READMEs](https://mermaid.stefs.me/){: .btn .btn-outline }
+[Creating Mermaid diagrams](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams){: .btn .btn-outline }
```

Comment:
May I suggest adding GitHub to the list (line 22) instead and removing completely this sentence instead?

Something like:
```
* [GitHub](https://github.com), see their documentation [Creating Mermaid diagrams](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams)
```